00-RELEASENOTES

```diff
@@ -11,6 +11,19 @@ CRITICAL: There is a critical bug affecting MOST USERS. Upgrade ASAP.
 SECURITY: There are security fixes in the release.
 --------------------------------------------------------------------------------
 
+================================================================================
+Valkey 7.2.11  -  Released TBD (not released yet)
+================================================================================
+
+Upgrade urgency SECURITY: This release includes security fixes we recommend you
+apply as soon as possible.
+
+Bug fixes
+=========
+
+* Ensure empty error tables in Lua scripts don't crash Valkey (#2229)
```

Comment:
`Prevent potential assertion in active defrag handling large allocations (#2353)`

I used this text for 8.1 somewhat to avoid the crash word. if you think yours (existing) is better I can revert mine.